common/components/renderCbs/UnitConverter.tsx

```diff
@@ -49,7 +49,13 @@ export class UnitConverter extends Component<Props, State> {
     });
   }
   private baseUnitCb = (value: string, decimal: number) => {
-    const baseUnit = toTokenBase(value, decimal).toString();
+    console.log('original value', value);
+    const parsedValue = parseDecimalNumber(value, '.,')
+      ? parseDecimalNumber(value, '.,').toString()
```

Comment:
You can avoid the double `toString()` calls here by calling `toString()` once on line 57